styles.css

```
@@ -2,7 +2,7 @@
 
 body {
     font-family: 'Slackey', cursive;
-    background-color: blue;
+    background-color: hsla(60, 100%, 50%, 0.434);
```

```suggestion
    background-color: #283943;
```

please use hex instead of hsla